job_templates/packaging_job.xml.em

```diff
@@ -32,11 +32,13 @@
     ignore_rmw_default=ignore_rmw_default,
     use_connext_debs_default=use_connext_debs_default,
 ))@
-        <hudson.model.BooleanParameterDefinition>
-          <name>CI_TEST_BRIDGE</name>
-          <description>By setting this to True, the tests for the ros1_bridge will be run.</description>
-          <defaultValue>@(test_bridge_default)</defaultValue>
```

Comment:
Since the use of the variable `test_bridge_default` is being removed it probably needs to be removed elsewhere too.